e2e/cypress/results/output.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="Mocha Tests" time="2.6440" tests="2" failures="0">
+  <testsuite name="Root Suite" timestamp="2021-08-20T21:35:35" tests="0" file="cypress/integration/public_access.spec.js" time="0.0000" failures="0">
+  </testsuite>
+  <testsuite name="Public Access" timestamp="2021-08-20T21:35:35" tests="2" time="2.6440" failures="0">
+    <testcase name="Public Access Cannot access the standard public pages" time="2.4520" classname="Cannot access the standard public pages">
+    </testcase>
+    <testcase name="Public Access Cannot access the dataset pages" time="0.1920" classname="Cannot access the dataset pages">
+    </testcase>
+  </testsuite>
+</testsuites>
```

Comment:
Is there any reason we want to track this file?

Reply:
I don't think so, removing and ignoring for the future.